### PR TITLE
[Stage of Stage] Add new constsants to help mitigate API changes

### PR DESCRIPTION
**Pull request by jfurrow**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @kennyt

File: src/js/constants/NodeStatuses.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
```
Comment (kennyt) [inline src/js/constants/NodeStatuses.js:1]: can we combine this file with `NodeStatusDisplayText` like the file `HealthBarStates` in dcos-ui?

**Comment by kennyt** [comment]
after the comment, i'll merge

File: src/js/constants/NodeStatuses.js (hunk 1 above)
Comment (jfurrow) [inline src/js/constants/NodeStatuses.js:1]: Good call, done!

**Pull request merged**